app/models/dinosaur.rb

```diff
@@ -1,6 +1,9 @@
 class Dinosaur < ActiveRecord::Base
+  belongs_to :user
+
   validates :name, presence: true
   validates :name, uniqueness: true
   validates :location_found, presence: true
   validates :info_url, presence: true
+  validates :user_id, presence: true
```

Comment:
Just validate against the `user`, because you care about the object itself, not merely whether its id is present.